index.js

```diff
@@ -0,0 +1,5 @@
+const server = require('./server.js');
+
+server.listen(4000, () => {
+  console.log('\n*** Server Running on http://localhost:4000 ***\n');
+});
```

Comment:
the next step in the interest of scalability is to store the port number in a constant outside this listen method (just `port` would be the simplest) and to make the logged message an interpolated string based on that variable, in case you need to quickly change the port value or if it needs to first check for potential environmental variables